mmcv/utils/parrots_jit.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+import torch
+
+TORCH_VERSION = torch.__version__
+
+if TORCH_VERSION == 'parrots':
+    from parrots.jit import pat
+    jit = pat
+else:
+
+    def jit(func=None,
+            check_input=None,
+            full_shape=True,
+            derivate=False,
+            coderize=False,
+            optimize=False):
+
+        def wrapper(func):
+
+            def wrapper_inner(*args, **kargs):
+                return func(*args, **kargs)
+
+            return wrapper_inner
+
+        if func is None:
+            return wrapper
+        else:
+            return func
+
+
+if TORCH_VERSION == 'parrots':
+    from parrots.utils.tester import skip_no_elena
+else:
+
+    def bypass_decorator(func):
```

Comment:
Can we simply use `def skip_no_elena(func)` here?

Reply:
At first, this flag comes from the cmake compilation option when compiling parrots, and then it is converted into a macro definition in C + +. Finally, it is transformed into an interface visible in Python through pybind11. I don't find a method to define this interface simply in mmcv.